src/main/java/com/capitalone/dashboard/repository/CommitRepository.java

```diff
@@ -31,7 +31,7 @@
 
     List<Commit> findByScmUrlIgnoreCaseAndScmBranch (String scmUrl, String scmBranch);
 
-    Commit findByScmRevisionNumberAndScmUrlIgnoreCaseAndScmBranchIgnoreCase (String scmRevisionNumber, String scmUrl, String branch);
+    List<Commit> findByScmRevisionNumberAndScmUrlIgnoreCaseAndScmBranchIgnoreCaseOrderByTimestampAsc (String scmRevisionNumber, String scmUrl, String branch);
```

Comment:
Is it not a breaking change? Rather add a new one like "findAllBy..."?

Reply:
I was quick to change the existing method since its only used on webhook. But yes, since core could have multiple consumers, it makes sense to add a new method, and keep the old one, thanks.